lib/trsort.c

```diff
@@ -328,13 +328,13 @@ tr_introsort(saidx_t *ISA, const saidx_t *ISAd,
              saidx_t *SA, saidx_t *first, saidx_t *last,
              trbudget_t *budget) {
 #define STACK_SIZE TR_STACKSIZE
-  struct { const saidx_t *a; saidx_t *b, *c; saint_t d, e; }stack[STACK_SIZE];
```

Comment:
Is stack[x].e dead code? If so this change alone is a win for cache efficiency unless their is some alignment reason why a dummy value was in there.